docs/source/running-autometa.rst

```diff
@@ -2,21 +2,15 @@
 Running Autometa
 ================
 
-Before running anything make sure you
-
-1. Running python3
```

Comment:
The reason specified `python3` was because Jason told me to explicitly tell users to use it. I guess he wanted that even a beginner can run it. But we can definitely remove it if your think it's unnecessary.

Reply:
To avoid confusion, we want the user to be in the appropriate python environment, ideally through the use of conda. When they are in this environment, they will need to call `python` so the other packages required are also in their environment. Specifying `python3` may result in unexpected behavior. We should write a comment in the same line though to make this explicit to the user